resources/state_table/huntsman.yaml

```diff
@@ -3,19 +3,26 @@ name: huntsman
 initial: sleeping
 location: states
 states:
-    - parked
-    - sleeping
-    - housekeeping
-    - ready
-    - calibrating
-    - scheduling
-    - preparing
-    - slewing
-    - focusing
-    - observing
-    - analyzing
-    - dithering
-    - parking
+    parked:
+        tags: always_safe
+    sleeping:
+        tags: always_safe
+    housekeeping:
+        tags: always_safe
+    ready:
+        tags: always_safe
+    calibrating:
+        tags: at_twilight
+    scheduling:
+        tags: always_safe
+    preparing:
+    slewing:
+    focusing:
```

Comment:
IIRC the plan was to enter `focusing` before astronomical twilight too, after `calibrating`.